thrift/atoms/tenfour.thrift

```diff
@@ -0,0 +1,61 @@
+namespace * tenfour
+
+include "../shared.thrift"
+
+/** date times are reprsented as i64 - epoch millis */
+typedef i64 DateTime
+
+struct Content {
```

Comment:
I don't know what `tenfour` but even if namespaced, I think `Content` is too generic and already used all over the place, so it will be very confusing.